src/migratus/database.clj

```diff
@@ -188,24 +197,24 @@
   [db modify-sql-fn table-name]
   (log/info "creating migration table" (str "'" table-name "'"))
   (let [timestamp-column-type (datetime-backend? db)]
-    (sql/with-db-transaction [t-con db]
-      (sql/db-do-commands
-        t-con
-        (modify-sql-fn
-          (sql/create-table-ddl table-name [[:id "BIGINT" "UNIQUE" "NOT NULL"]
-                                            [:applied timestamp-column-type "" ""]
-                                            [:description "VARCHAR(1024)" "" ""]]))))))
+    (jdbc/with-transaction [t-con (connection-or-spec db)]
+      (jdbc/execute!
+       t-con
+       (modify-sql-fn
+        (str "CREATE TABLE " table-name 
```

Comment:
We need to sanitize `table-name` here as well.

Reply:
done.